CMakeLists.txt

```diff
@@ -6,6 +6,10 @@ if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.30.0)
   cmake_policy(SET CMP0167 OLD)
 endif()
 
+# use gnu standard install directories
+include(GNUInstallDirs)
+set(ARBORX_INSTALL_PACKAGEDIR "${CMAKE_INSTALL_DATADIR}/cmake/ArborX" CACHE PATH "Install location of CMake target files")
```

Comment:
`ARBORX_INSTALL_PACKAGEDIR` is a replacement for the hard-coded `lib/cmake/ArborX` in a few place. And as there is nothing in `ArborXSettings.cmake.in` that is really arch dependent I made the default `${CMAKE_INSTALL_DATADIR}/cmake/ArborX` instead of `${CMAKE_INSTALL_LIBDIR}/cmake/ArborX`, but we could leave the default untouched.

Reply:
I would have stuck to the default

Reply:
Changed, it is configurable now, no big deal.